CONTRIBUTING.md

```diff
@@ -58,7 +79,7 @@ We have very precise rules over how our git commit messages should be formatted.
 Should be `docs`
 
 #### Scope
-The scope can be anything specifying the place of the commit change. For example `router`, `prerendering`, etc. If you make multiple commits for the same doc, please keep the naming of this doc consistent. For example, if you make a change to the router docs and the first commit is `docs(router)`, you should continue to use `router` for any more commits related to navigation.
+The scope can be anything specifying the place of the commit change. For example `props`, `angular`, etc. If you make multiple commits for the same doc, please keep the naming of this doc consistent. For example, if you make a change to the `@Prop` docs and the first commit is `docs(prop)`, you should continue to use `prop` for any more commits related to navigation.
```

Comment:
I think we're missing something here. The last sentence doesn't make much sense 🤔

Reply:
Yeah, TBH I think we can remove those last two

Reply:
https://github.com/ionic-team/stencil-site/pull/1052/commits/9d930c7760887ffbc524648aab27677abbfb9e2d gets rid of that weird wording

Reply:
That feels better. Nice and simple